src/api/app/models/tagcloud.rb

```diff
@@ -9,7 +9,7 @@ class Tagcloud
   def initialize(opt)
     ApplicationRecord.logger.debug "[TAG:] building a new tag cloud."
 
-    @limit = opt[:limit] or @limit = 0
+    (@limit = opt[:limit]) || (@limit = 0)
```

Comment:
I had seen it :joy: I didn't change to avoid making the review too complicate. I'll add it in a different  commit with other style changes not really related to the `AddOr` cop.